presto-orc/src/main/java/com/facebook/presto/orc/StreamId.java

```diff
@@ -50,7 +48,7 @@ public StreamKind getStreamKind()
     @Override
     public int hashCode()
     {
-        return Objects.hash(column, streamKind);
+        return 31 * column + streamKind.hashCode();
```

Comment:
For some files, yes.  We have a few degenerate files that have thousands of 100 row stripes so the stripe setup dominates performance.  This method is used everywhere in setup and it causes boxing and array creation.   For normal files, stripe setup is tiny compared to the data processing.